packages/common/src/models/Track.ts

```diff
@@ -74,12 +75,18 @@ export type PremiumConditionsSolNFTCollection = {
   externalLink: Nullable<string>
 }
 
+export type PremiumConditionsUSDCPurchase = {
+  price: StringWei
```

Comment:
Worth discussing - I think we can have this be `StringUSDC` and pad it to the correct number of decimals on DN. We added extra padding to WAudio on DN because we combine the Eth + Sol AUDIO currencies together in the client, and thus need the same amount of precision. But we will never show USDC combined, so should be able to index it without extra padding. cc @rickyrombo

Reply:
I was wondering how many decimals USDC has in both Ethereum and Solana land so I did some poking around, and it seems like it's 6 decimals in both!

Reply:
Oh this makes sense. I remember now that AUDIO has different precision in the two environments, while USDC does not. I agree that we can/should just make this `StringUSDC`